### PR TITLE
Added needed grunt commands to create a artifact 

### LINKED ISSUE #365: Create grunt commands to generate a artifact zip
In Yoast SEO we have grunt commands, like grunt update-version, grunt copy:artifact and grunt compress:artifact which updates then version in the relevant files. This needs to be present in Yoast SEO news too.

### PR DESCRIPTION
**Pull request by dutchmartin**
## Summary

This PR can be summarized in the following changelog entry:

* Adds `copy:artifact` and `compress:artifact` command to Grunt to create an artifact zip.

## Relevant technical choices:

* Using the grunt compress package
* Using the grunt copy package
* Adding a command that builds the project and creates a zipped artifact using the above named packages

## Test instructions

This PR can be tested by following these steps:

1. Ensure you have all development dependency's for the project by running `yarn install`
2.  Execute the new command `create:artifact`, this is an alias that contains the following commands: 
    * `update-version`
    *  `build`
    * `copy:artifact` 
    * `compress:artifact`
The last 2 of this list are new commands
3. Check if the command does what is described below
`create:artifact: Create a zipped artifact from a fresh build`
`copy:artifact: Copy the needed files to the ./artifact folder`
`compress:artifact: Create a compressed zip from the ./artifact folder`

Fixes #365

### CODE REVIEW

File: grunt/config/copy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	artifactFiles: {
+		files: [
+			{
+				expand: true,
+				cwd: ".",
+				src: [
+					// folders to copy
+					"classes/**",
+					"vendor/**",
+					"assets/**",
```
Comment (moorscode) [inline grunt/config/copy.js]: The non-minified JavaScript files should not be added to the artifact.

**Comment by jcomack** [comment]
CR done 👍

**Comment by jcomack** [comment]
Acceptance failed 🚧 

I attempted to follow the testing steps, however this already fails on step 2, command 1 as there is nothing to compress. Please fix the testing steps to properly reflect in what order things should be tested manually and how the entire process can be tested in one go.

**Comment by jcomack** [comment]
CR done 👍

**Comment by jcomack** [comment]
Acceptance done 👍

**Pull request merged**